apkid/rules/apk/obfuscators.yara

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2018  RedNaga. https://rednaga.io
+ * All rights reserved. Contact: rednaga@protonmail.com
+ *
+ *
+ * This file is part of APKiD
+ *
+ *
+ * Commercial License Usage
+ * ------------------------
+ * Licensees holding valid commercial APKiD licenses may use this file
+ * in accordance with the commercial license agreement provided with the
+ * Software or, alternatively, in accordance with the terms contained in
+ * a written agreement between you and RedNaga.
+ *
+ *
+ * GNU General Public License Usage
+ * --------------------------------
+ * Alternatively, this file may be used under the terms of the GNU General
+ * Public License version 3.0 as published by the Free Software Foundation
+ * and appearing in the file LICENSE.GPL included in the packaging of this
+ * file. Please visit http://www.gnu.org/copyleft/gpl.html and review the
+ * information to ensure the GNU General Public License version 3.0
+ * requirements will be met.
+ *
+ **/
+
+include "common.yara"
+
+rule arxan_guardit : obfuscator
+{
+  meta:
+    description = "Arxan GuardIT"
+    url         = "https://www.arxan.com"
+    sample      = ""
```

Comment:
Is the sample hash private? If you send me the sample privately, I can find a public hash.

Reply:
Done!